crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -517,24 +532,81 @@ type NameServiceEdge {
 }
 
 type Object implements ObjectOwner {
+	"""
+	If this object is a MoveObject, this is a number that increases each time a tx takes this object as a mutable input
+	This is a lamport timestamp, not a sequentially increasing version
+	If this object is a MovePackage, the version is increased every time the package is upgraded
+	Note that packages are referred to by move calls using just their ID, and they are always loaded at their latest version
+	"""
 	version: Int!
+	"""
+	The object digest is a 32-byte hash of the sui_type::Object struct
+	This uniquely identifies the object
+	"""
 	digest: String!
+	"""
+	The amount of SUI we would rebate if this object gets deleted
+	This number is recalculated each time the object is mutated based on
+	the present storage gas price
+	"""
 	storageRebate: BigInt
+	"""
+	This is the Base64 encoded bcs serialization of the sui_type::Object struct
+	"""
 	bcs: Base64
+	"""
+	This is the transaction block that created or most recently mutated this object
+	"""
 	previousTransactionBlock: TransactionBlock
+	"""
+	Objects can either be immutable, shared, owned by another address,
+	or are the dynamic child objects of a parent object
```

Comment:
There are child objects and there are dynamic (object) fields and the two are related, but there are no dynamic child objects.

```suggestion
	Objects can either be immutable, shared, owned by an address,
	or are child objects (part of a dynamic field)
```